CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ endif("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
 # BUILD OPTIONS
 option(NETWORKIT_BUILD_CORE "Build NetworKit core library" ON)
 option(NETWORKIT_BUILD_TESTS "Build NetworKit C++ tests" OFF)
-option(NETWORKIT_LOGGING "Build with logging support" ON)
```

Comment:
Should we first mark `NETWORKIT_LOGGING` as DEPREACTED and issue an error message hinting to the new options if it is ever set to OFF?

Reply:
I am not sure if it's worthwhile to do this. The option is not used in any standard build (e.g., `setup.py`) and it is quite easy to adapt to the new option. If you think that it is worthwhile, I suggest to fail the build if the option is specified so that users are forced to adapt their builds.